src/components/MintSong/MintSong.tsx

```diff
@@ -132,7 +131,7 @@ const MintSong: React.FC = () => {
         isMinting={isMinting}
         isVisible={showModal}
         onHandleModal={toggleModal}
-        onHandleMintForm={handleMintForm}
+        onHandleMintForm={(d) => handleMintForm(d)}
```

Comment:
What is `d` supposed to represent? data? Could you please make it more explicit

Reply:
It is just a wrapper for handleMint, so the smaller looks nice since it does nothing than calling other function

Reply:
It's better to be long and clear than to be concise and confusing Om. If leaving it there is actually important, then we should please name it properly.